tests/AL01/Dataset/BC001_Alignment.xml

```diff
@@ -0,0 +1,2078 @@
+                    <Start>1253540.84464 2691506.87248</Start>
+                    <Center>1252181.390894 2690849.616432</Center>
+                    <End>1253521.518594 2691545.430882</End>
+                </Curve>
+                <Spiral length="50.000010" radiusEnd="INF" radiusStart="1510.000000" rot="cw" spiType="clothoid" constant="274.772661" dirEnd="4.2169314432" dirStart="4.2334877379" theta="0.0165562947" totalY="0.275933" totalX="-49.998639" staStart="13089.278460" tanLong="33.333819" tanShort="16.667105">
+                    <Start>1253521.51859 2691545.43088</Start>
+                    <PI>1253513.838316 2691560.222965</PI>
+                    <End>1253497.99028 2691589.54843</End>
+                </Spiral>
+                <Line dir="4.2169313875" length="157.389290" staStart="13139.278470">
+                    <Start>1253497.99028 2691589.54843</Start>
+                    <End>1253423.162018 2691728.011854</End>
+                </Line>
+                <Line dir="4.2169320158" length="352.382940" staStart="13296.667760">
+                    <Start>1253423.1618 2691728.01174</Start>
+                    <End>1253255.627069 2692038.021176</End>
+                </Line>
+                <Spiral length="127.998530" radiusEnd="740.000000" radiusStart="INF" rot="ccw" spiType="clothoid" constant="307.764378" dirEnd="4.3034308608" dirStart="4.2169453676" theta="0.0864854932" totalY="3.688034" totalX="127.902824" staStart="13649.050700" tanLong="85.365810" tanShort="42.696594">
+                    <Start>1253255.62707 2692038.02118</Start>
+                    <PI>1253215.042288 2692113.122425</PI>
+                    <End>1253198.06383 2692152.29807</End>
+                </Spiral>
+                <Curve rot="ccw" chord="66.250015" crvType="arc" dirEnd="4.3929896009" dirStart="4.3034326279" radius="740.000000" length="66.272160" staStart="13777.049230">
+                    <Start>1253198.06383 2692152.29807</Start>
+                    <Center>1253877.040715 2692446.560521</Center>
+                    <End>1253174.46682 2692214.203204</End>
+                </Curve>
+                <Spiral length="103.023610" radiusEnd="2600.000000" radiusStart="740.000000" rot="ccw" spiType="clothoid" constant="326.448324" dirEnd="4.4824134180" dirStart="4.3929906380" theta="0.0894227800" totalY="3.749043" totalX="-102.920318" staStart="13843.321390" tanLong="61.107204" tanShort="41.980851">
+                    <Start>1253174.46683 2692214.20321</Start>
+                    <PI>1253161.285036 2692254.060858</PI>
+                    <End>1253147.355411 2692313.559244</End>
+                </Spiral>
+                <Spiral length="82.488820" radiusEnd="2600.000000" radiusStart="2600.000000" rot="ccw" spiType="clothoid" constant="0.000000" dirEnd="4.5141308569" dirStart="4.4824043877" theta="0.0317264692" totalY="1.308430" totalX="-82.474982" staStart="13946.345000" tanLong="41.247870" tanShort="41.247870">
```

Comment:
How can the start radius and the end radius be the same for a transition spiral?
radiusEnd="2600.000000" radiusStart="2600.000000"

Reply:
@BenzclyZhang can you please check this out? Thanks

Reply:
This segment is wrong. It is removed.